### PR TITLE
PR for columns_ui#59

### PR DESCRIPTION
**Pull request by MAxonn**
Hey Reupen. I'm back from holiday and here's the promised PR :).

### CODE REVIEW

**Comment by reupen** [comment]
Hi,

That looks like it should do the trick :)

My main comment would be to assign GetSysColor(COLOR_BTNTEXT) to a (const) variable before the first use of it, and then use that variable in the calls to uih::text_out_colours_tab(). Mainly to avoid repetition.

I'm quite happy to use auto so you could go for:
```
const auto default_text_colour = GetSysColor(COLOR_BTNTEXT);
```

**Comment by MAxonn** [comment]
Good point :). A few extra bytes in the memory are worth saving 5 calls. I'll make the change tomorrow and resubmit. Gotta catch some Zs now :).

**Comment by MAxonn** [comment]
It is done :).

Also, I'm planning to work with a few more things in Columns UI. Could I ask you a few questions about that? It also works if I post these questions on the hydrogen audio forum and then send you a link somehow, if you prefer to keep all your answers documented there (you already answered to some questions of mine there some time ago, but I didn't have time to do much back then).

File: foo_ui_columns/status_pane_msgproc.cpp [hunk 1]
```diff
@@ -89,23 +89,24 @@ LRESULT status_pane::on_message(HWND wnd,UINT msg,WPARAM wp,LPARAM lp)
             HFONT fnt_old = SelectFont(dc, m_font);
 
             const char * placeholder = "999999999 items selected";
+						const auto default_text_colour = GetSysColor(COLOR_BTNTEXT);
```
Comment (reupen) [inline foo_ui_columns/status_pane_msgproc.cpp]: Looks like some tabs have crept in – this repo uses spaces for indentation so the tabs will need replacing with spaces.

**Comment by reupen** [comment]
@MAxonn If they're about specific bugs or new features etc., you can create issues here and put any questions in them. If you don't have an issue in mind to create, then the forums is fine.

**Comment by MAxonn** [comment]
Done again :).

**Pull request merged**